app/src/main/res/values/strings.xml

```diff
@@ -192,6 +192,7 @@
   <string name="location_permission_title">Requesting Location Permission</string>
   <string name="ok">OK</string>
   <string name="warning">Warning</string>
+  <string name="duplicate_file_name">Duplicate File Name found</string>
   <string name="duplicate_image_found">Duplicate Image Found</string>
```

Comment:
Is this duplicate_image_found still used somewhere?

Reply:
Nope.

Reply:
Seems like we can delete it

Reply:
Please remove it then 🙂

Reply:
Done!